### DISCUSSION TITLE
Hide or move the stress value  on the NMDS plot

### DISCUSSION THREAD

**Discussion by JUTU568** [ChiLiubio/microeco#381] [Q&A] [answered]
Hi, first of all, thank you so much for this detailed tutorial! 
When I was making an NMDS plot through "cal_ordination(ordination = "NMDS")" , the stress value stored in t1$res_ordination is always shown on the plot (random position). I am wondering how can I remove the stress value (or avoid it from being generated on the plot). Or how can I move the position of it? Thank you so much!

**Comment by ChiLiubio** [comment, marked as answer]
Hi. The parameter `NMDS_stress_pos` in plot_ordination function can adjust the position. When it is NULL, no text will be show. Please see the help document for the explanations. 
![图片](https://github.com/ChiLiubio/microeco/assets/20815519/60b690af-6eaa-4f8f-ab7e-984cf5d941d1)
